tools/publisher/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
assuming this commit is straight copy/paste I did not review it

Reply:
That's correct.